src/main/kotlin/uk/gov/justice/hmpps/offendersearch/config/SecurityUserContext.kt

```diff
@@ -1,40 +1,16 @@
 package uk.gov.justice.hmpps.offendersearch.config
 
-import org.springframework.security.core.Authentication
 import org.springframework.security.core.context.SecurityContextHolder
-import org.springframework.security.core.userdetails.UserDetails
 import org.springframework.stereotype.Component
+import uk.gov.justice.hmpps.offendersearch.security.AuthAwareAuthenticationToken
 
 @Component
 class SecurityUserContext {
-  private val authentication: Authentication
-    get() = SecurityContextHolder.getContext().authentication
+  val authentication: AuthAwareAuthenticationToken?
+    get() = SecurityContextHolder.getContext().authentication as AuthAwareAuthenticationToken
 
```

Comment:
the only scenario this API is used is with this type of token, removed all the extra stuff that was copied from hmpss auth not required